Source/WebInspectorUI/UserInterface/Views/NetworkResourceDetailView.js

```diff
@@ -121,7 +121,9 @@ WI.NetworkResourceDetailView = class NetworkResourceDetailView extends WI.Networ
         switch (identifier) {
         case "preview":
             if (!this._resourceContentView)
-                this._resourceContentView = this._contentBrowser.showContentViewForRepresentedObject(this.representedObject);
+                this._resourceContentView = this._contentBrowser.showContentViewForRepresentedObject(
+                    this.representedObject, false, {inNetworkView: true}
```

Comment:
style: please create a `const cookie = null;` variable instead of inlining `false`
style: please put all of this back on one line
```
if (!this._resourceContentView) {
    const cookie = null;
    this._resourceContentView = this._contentBrowser.showContentViewForRepresentedObject(this.representedObject, cookie, {disableDropZone: true});
}
```